File: Daily Challenge/Java/Ascending, Descending Or Equal 2.7.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] a = new int[n];
        boolean eq = true, asc = true;

        for(int i = 0; i < n; i++)
            a[i] = sc.nextInt();

        for(int i = 1; i < n; i++) {
            if(a[i] != a[0]) eq = false;
            if(a[i] < a[i - 1]) asc = false;
        }

        if(eq)
            System.out.println("Equal");
        else if(asc)
            System.out.println("Ascending");
        else
            System.out.println("Descending");
    }
}

File: Daily Challenge/C/Ascending, Descending Or Equal 2.7.2025.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    int a[n], i, eq = 1, asc = 1;

    for(i = 0; i < n; i++)
        scanf("%d", &a[i]);

    for(i = 1; i < n; i++) {
        if(a[i] != a[0]) eq = 0;
        if(a[i] < a[i-1]) asc = 0;
    }

    if(eq)
        printf("Equal");
    else if(asc)
        printf("Ascending");
    else
        printf("Descending");

    return 0;
}